consensus/poet/core/sawtooth_poet/poet_consensus/consensus_state.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+from collections import namedtuple
+
+import cbor
+
+ValidatorState = \
+    namedtuple(
+        'ValidatorState',
+        ['key_block_claim_count',
+         'poet_public_key',
+         'total_block_claim_count'
```

Comment:
Do you want to store a block depth since registration for C and K tests?
I believe key_block_claim_count is for Z test. 
I'm no sure what total_block_claim_count is used for?